source/reference/limitations.txt

```diff
@@ -171,3 +171,18 @@ System Collections
 ------------------
 
 .. include:: /includes/collections/behavior-system-collections.rst
+
+Rolling Index Builds
+--------------------
+
+``mongosync`` does not sync source cluster indexes that use a `rolling
+index build
+<https://www.mongodb.com/docs/manual/tutorial/build-indexes-on-replica-sets/>`_.
```

Comment:
The `:ref:` role can find targets on other repos, so we should use that.

If we rename the server file or delete it, this link won't generate an error until the user 404's. A `:ref:` will follow moves and start throwing build errors if we remove the file.

Reply:
Fixed!